src/main/java/uk/gov/hmcts/sptribs/CaseApiApplication.java

```diff
@@ -42,7 +43,8 @@
         ExtendedCaseDataApi.class,
         CaseAssignmentApi.class,
         CaseDocumentClientApi.class,
-        BundlingClient.class
+        BundlingClient.class,
+        LinkReasonClient.class
```

Comment:
Doesn't seem to be related to case flag feature.

Reply:
Yes, added as part of merge. Will remove this change